packages/expo-web-browser/src/WebBrowser.types.ts

```diff
@@ -124,6 +131,48 @@ export enum WebBrowserResultType {
   LOCKED = 'locked',
 }
 
+// @needsAudit
+/**
+ * A browser presentation style. Its values are directly mapped to the [`UIModalPresentationStyle`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621355-modalpresentationstyle).
+ *
+ * @platform ios
+ */
+export enum WebBrowserPresentationStyle {
+  /**
+   * A presentation style in which the presented browser covers the screen.
+   */
+  FULL_SCREEN = 'fullScreen',
+  /**
+   * A presentation style that partially covers the underlying content.
+   */
+  PAGE_SHEET = 'pageSheet',
+  /**
+   * A presentation style that displays the browser centered in the screen.
+   */
+  FORM_SHEET = 'formSheet',
+  /**
+   * A presentation style where the browser is displayed over the app's content.
+   */
+  CURRENT_CONTEXT = 'currentContext',
+  /**
+   * A presentation style in which the browser view covers the screen.
+   */
+  OVER_FULL_SCREEN = 'overFullScreen',
+  /**
+   * A presentation style where the browser is displayed over the app's content.
+   */
+  OVER_CURRENT_CONTEXT = 'overCurrentContext',
+  /**
+   * A presentation style where the browser is displayed in a popover view.
+   */
+  POPOVER = 'popover',
+  /**
+   * **iOS 13+:** The default presentation style chosen by the system.
+   * On older iOS versions, falls back to `WebBrowserPresentationStyle.FullScreen`.
```

Comment:
```suggestion
   * @platform ios 13+ 
   * The default presentation style chosen by the system.
   * On older iOS versions, falls back to `WebBrowserPresentationStyle.FullScreen`.
```

Let's use annotation to mark required iOS version.

Reply:
Thanks! I missed that point in the `Expo modules style guide` 😅